pkg/gomega/matchers/contain_item_with_prefix.go

```diff
@@ -35,3 +35,32 @@ func ContainItemWithPrefix(prefix string) types.GomegaMatcher {
 		return false, nil
 	})
 }
+
+// ContainItemWithOwnerWithPrefix is a gomega matcher that can be used to assert that a
+// Kubernetes list object contains an item olm.owner label with the provided prefix
+// This matcher is preferred in case of operator related objects when object name itself does not contain desired prefix,
+// but has olm.owner label which contains it
```

Comment:
Can you run gofmt on this file? I feel like this comment should be formatted differently, maybe not